drivers/CoreSPI/source/CoreSPI.cpp

```diff
@@ -0,0 +1,18 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreSPI.h"
+
+using namespace leka;
+
+auto CoreSPI::write(int value) -> int
+{
+	return _spi.write(value);
+}
+
+auto CoreSPI::write(const uint8_t *tx_buffer, int tx_length, uint8_t *rx_buffer, int rx_length) -> int
+{
+	return _spi.write(reinterpret_cast<const char *>(tx_buffer), tx_length, reinterpret_cast<char *>(rx_buffer),
```

Comment:
rajouter le commentaire suivant:

```suggestion
	// ? NOLINTNEXTLINE - allow reinterpret_cast as there are no alternatives
	return _spi.write(reinterpret_cast<const char *>(tx_buffer), tx_length, reinterpret_cast<char *>(rx_buffer),
```